reference/index.md

```diff
@@ -486,3 +486,59 @@ __URL parameters:__
 {% highlight json %}
 {}
 {% endhighlight %}
+
+## Trackers
+Trackers make it possible to track a shipment that wasn't creating using shipcloud. They are
+basically a way to monitor shipments created elsewhere. All you have to do is provide us with the
+tracking number you received from the carrier as well as its corresponding name acronym.
+
+{% include reference/trackers_request_togglebox.html %}
+
+### Creating a tracker
+
+This is the way to create a tracker based on a tracking number provided by one of the carriers
```

Comment:
Maybe we should specify `"one of the carriers"` and limit them to carriers supported by shipcloud? What do you think?